src/Oci8/Oci8Connection.php

```diff
@@ -367,6 +368,16 @@ protected function getConfigSchemaPrefix()
         return isset($this->config['prefix_schema']) ? $this->config['prefix_schema'] : '';
     }
 
+    /**
+     * Get config max length.
+     *
+     * @return string
+     */
+    protected function getConfigMaxLength()
+    {
+        return isset($this->config['max_name_len']) ? $this->config['max_name_len'] : 30;
```

Comment:
Maybe something like on PR #768 

```php
$version = (int) filter_var($this->connection->getConfig('server_version'), FILTER_SANITIZE_NUMBER_INT);

return $version >= 12 ? 126 : 30;
```

Reply:
So, somerhing like this ?
```php
$version = (int) filter_var($this->config['server_version'], FILTER_SANITIZE_NUMBER_INT);
return $version >= 12 ? $this->config['max_name_len'] ?? 30 : 30;
```

Reply:
No need for `max_name_len`. I think 126 is the new constant length for 12 and above.

Reply:
We can also remove the max_name_len config added in the PR.

Reply:
> No need for `max_name_len`. I think 126 is the new constant length for 12 and above.
For me is ok, but they change the limit in oracle 12.2.0.1 if we compare with 12 some people can have a problem if they have 12.1.0.1

Reply:
Updated the suggestion to reflect your code on config.

Reply:
That makes sense. Also reading the new max length, is 128 per [SO post](https://stackoverflow.com/questions/756558/what-is-the-maximum-length-of-a-table-name-in-oracle)

> In [Oracle 12.2](http://docs.oracle.com/database/122/REFRN/ALL_OBJECTS.htm#REFRN20146) and above the maximum object name length is 128 bytes.
> In [Oracle 12.1](https://docs.oracle.com/database/121/SQLRF/sql_elements008.htm#SQLRF00223) and below the maximum object name length is 30 bytes.